apps/netlify/frontend/src/sidebar/build-button.js

```diff
@@ -94,7 +94,7 @@ export default class NeflifySidebarBuildButton extends React.Component {
       userId: this.props.userId,
     });
 
-    const { buildHookId } = this.props.site;
+    const buildHookId = '61e9633db95ff20fe700baa1';
```

Comment:
Is this leftover? 👀

Reply:
Good catch 🤦🏾. From testing things using the frontend app